README.md

```diff
@@ -5,3 +5,9 @@ This guide covers the necessary bits. As the project evolves, it will only becom
 ## Updating
 
 This site uses the Compose Hugo theme loaded as a Hugo module. Updating the theme from the base site directory: `hugo mod get -u ./...`
+
+To run a live server as you're working on doc updates, please use the following command:
+
+```
+hugo server --bind 0.0.0.0 -b `hostname -f`
```

Comment:
You really don't need anything past `hugo server` unless you specifically need to bind to all interfaces (useful when allowing another device to access the live server instance).